stories/main.stories.tsx

```diff
@@ -1,12 +1,28 @@
 import type { Meta, StoryObj } from "@storybook/react";
 import { FormBuilder } from "src";
 import { DocElement } from "./doc-elements";
+import { TElement } from "src/context-providers";
+import { action } from "@storybook/addon-actions";
 
 type Component = typeof FormBuilder;
 
 const meta: Meta<Component> = {
     title: "Main",
     component: DocElement,
+    /**
+     * Below is a workaround for a type error in the storybook addon when using an async function.
+     * https://github.com/storybookjs/storybook/issues/23731
+     */
+    args: {
+        onSubmit: async (...args) => {
+            action("onSubmit")(args);
+        },
+    },
```

Comment:
I dont see any type errors even after commenting this block of code, is it still present?